openedx/features/course_experience/static/course_experience/js/course_outline_factory.js

```diff
@@ -23,6 +24,16 @@
                         break;
                     }
                 });
+
+                $('a:not([href^="#"])').click(function(event) {
```

Comment:
@bjacobel @andy-armstrong: This event (`'edx.ui.lms.link_clicked'`) is so generically named.  Are we supposed to be tracking every link click in LMS?  If so, is there a more generic place we could add this so if doesn't need to go in each individual factory?